ResearchKit/Common/ORKChoiceAnswerFormatHelper.m

```diff
@@ -71,6 +75,14 @@ - (NSUInteger)choiceCount {
     return _choices.count;
 }
 
+- (id<ORKAnswerOption>)answerOptionAtIndex:(NSUInteger)index {
+    if (index >= _choices.count) {
+        return nil;
+    }
+    
+    return _choices[index];
+}
+
 - (ORKImageChoice *)imageChoiceAtIndex:(NSUInteger)index {
     if (index >= _choices.count) {
```

Comment:
It would nice to let `imageChoiceAtIndex:` and `textChoiceAtIndex:` reuse `answerOptionAtIndex:`